birthdays-project/BirthdayApp/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+
+class Birthdate(models.Model):
+    dob = models.DateField(max_length=8)
+
+    # Displays date in a dd-MMM-yy format.
+    def __str__(self):
+        return '{:%d %b %y}'.format(self.dob)
+
+
+# Creates a user Model.
+class User(models.Model):
+    firstName = models.TextField(max_length=20, verbose_name='First Name')
```

Comment:
I've never liked underscores and have always preferred camel casing, but I'll adapt. :) CharFields were way better for the task. I was able to delete the code modifying the admin side as well since the boxes were no longer huge TextFields.